interfaces/builtin/fwupd.go

```diff
@@ -55,6 +56,9 @@ const fwupdPermanentSlotAppArmor = `
   # Allow libfwup to access efivarfs with immutable flag
   capability linux_immutable,
 
+  capability dac_read_search,
+  capability dac_override,
```

Comment:
Do you need to do anything for `SG_IO` ioctl interface?

Reply:
`HIDIOCSFEATURE` and `HIDIOCGFEATURE` for Elan plugin, anything needed for those?

Reply:
`MMC_IOC_CMD` and `MMC_IOC_MULTI_CMD` for eMMC plugin

Reply:
Would not it be covered by  "capability sys_rawio"? Maybe I can group the capabilities together in the file.

Reply:
> Would not it be covered by  "capability sys_rawio"? Maybe I can group the capabilities together in the file.

I'm not familiar with the capability, but I figured I should suggest the specific ioctl used so you can confirm they're covered or anything else needed.

Reply:
Looking at the kernel...

`SIOCETHTOOL` looks like needing `CAP_NET_ADMIN`.

I am guessing `SG_IO` needs `CAP_SYS_RAWIO`.

I do not think we need any capability for `HIDIOCSFEATURE` and `HIDIOCGFEATURE`.

`MMC_IOC_CMD` and `MMC_IOC_MULTI_CMD` probably need `CAP_SYS_RAWIO`.

`NVME_IOCTL_ADMIN_CMD` will need `CAP_SYS_ADMIN`.

Reply:
Added `CAP_NET_ADMIN` and commented on the different capabilities.